src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarMetadata.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Box } from "Styleguide/Elements/Box"
+import { ArtworkSidebarClassificationFragmentContainer as Classification } from "./ArtworkSidebarClassification"
+import { ArtworkSidebarSizeInfoFragmentContainer as SizeInfo } from "./ArtworkSidebarSizeInfo"
+import { ArtworkSidebarTitleInfoFragmentContainer as TitleInfo } from "./ArtworkSidebarTitleInfo"
+
+import { ArtworkSidebarMetadata_artwork } from "__generated__/ArtworkSidebarMetadata_artwork.graphql"
+
+export interface ArtworkSidebarMetadataProps {
+  artwork: ArtworkSidebarMetadata_artwork
+}
+
+const ArtworkSidebarMetadataContainer = Box
```

Comment:
How come you decided to keep it here?

Reply:
Good catch! It’s solely there because I missed it.